### PR TITLE
Bionic Descriptions Update (Pt. One)

### PR DESCRIPTION
**Pull request by DukePaulAtreid3s**
Part one

#### Summary
Category "Brief description"
Updating the descriptions on many of the the existing CBMs to be more in-line with the descriptions of the Exodii weapons.

#### Purpose of change

The Exodii guns often have fun, interesting descriptions of the History of the Weapon as well as its effects. However, the CBMs, also produced by the Exodii, have quite plain, boring descriptions.

#### Describe the solution
Editing the descriptions of many Bionics to make them more interesting, while still keeping all of the  useful information.

#### Describe alternatives you've considered
Not doing this

### CODE REVIEW

**Comment by Karol1223** [comment]
I am really not a fan of this. Adding some alternate reality speculative lore for dimensions the player wouldn't know about feels very wrong.

**Comment by KeremBabaG** [comment]
> I am really not a fan of some of this. Adding some alternate reality speculative lore for dimensions the player wouldn't know about feels very wrong. The ones that just describe the item are fine, but the lore gimmicks feel off...

we have the same for exodii weapons, i think this one is fine

**Comment by Karol1223** [comment]
> we have the same for exodii weapons, i think this one is fine

For the record, my opinion of these for the weapons was and still is the exact same.

File: data/json/items/bionics.json [hunk 1]
```diff
@@ -258,7 +258,7 @@
     "type": "BIONIC_ITEM",
     "name": { "str": "Cloaking System CBM" },
     "looks_like": "bio_int_enhancer",
-    "description": "A high-power system that uses a set of cameras and LEDs to make the user blend into their surroundings, rendering them fully invisible to eyes and optical sensors.  However, this doesn't prevent detection from other means such as infrared, sonar, etc.",
+    "description": "A high-power system that uses a set of cameras and LEDs to make the user blend into their surroundings, rendering them fully invisible to eyes and optical sensors.  However, this doesn't prevent detection from other means such as infrared, sonar, etc. Their manufacturer, Khan Defense Solutions, faced several lawsuits concerning mental illness stemming from extended use. However, these claims were litiated by the onset of rapture.",
```
Comment (Procyonae) [inline data/json/items/bionics.json:261]:
```suggestion
    "description": "A high-power system that uses a set of cameras and LEDs to make the user blend into their surroundings, rendering them fully invisible to eyes and optical sensors.  However, this doesn't prevent detection from other means such as infrared, sonar, etc.  Their manufacturer, Khan Defense Solutions, faced several lawsuits concerning mental illness stemming from extended use.  However, these claims were overruled by the onset of rapture.",
```
I don't think you can use litigated like that, went with overruled as a still silly but more sensical alternative, using something not silly may be advisable instead tho

**Review by Procyonae — CHANGES_REQUESTED** [review]
You need two spaces between every sentence for Transifex the translation thing we use

**Comment by fairyarmadillo** [comment]
I don't like these either. It's a bunch of cruft on items that are often pretty confusing to begin with, it's information the player character would have no way of knowing, and implies a bunch of zany sci-fi lore that reads as very unserious, which hardly seems appropriate for the faction.

**Comment by Procyonae** [comment]
I'd prefer if an exodii could tell you stuff about their items upon presenting them instead of this stuff being in the description but I don't see any harm in the mean time

**Comment by fairyarmadillo** [comment]
> I'd prefer if an exodii could tell you stuff about their items upon presenting them instead of this stuff being in the description but I don't see any harm in the mean time

The harm is that it's now establishing them as a wacky faction and canonizing a bunch of weird stuff that shouldn't be lore.

"Bare Grills"? Seriously? Is that guy even on TV anymore? Why is the blood analyzer referencing Greek mythology? Why are the bionic eyes making reference to Apple products? Do we have any confirmation whatsoever that they were ever an interplanetary civilization? That brings up a bunch of questions about why they're on Earth and not, like, the moon or mars or some other place which would be much safer.

**Comment by DukePaulAtreid3s** [comment]
> obviously.
> 
> And do we have any confirmation whatsoever that they were ever an interplanetary civilization? That brings up a bunch of questions about why they're on Earth and not, like, the moon or Mars or some other place which would be much safer.

You make some good points, and I will adjust some of them accordingly. However, in lore, many CBMs aren't manufactured by the Exodii, but are bionics that were picked up by the Exodii and then unified under the CBM system. Once again, 100% agree that "Bare Grills" isn't my finest moment. I just want to spice some of them up, and this WIP.

**Comment by DukePaulAtreid3s** [comment]
Actually, I just realize I may be fundamentally misunderstanding the Exodii lore so I am closing this until I come back with better tone.

**Pull request closed without merging**